Dashboard/DockerContext/dashboard/constants.py

```diff
@@ -30,85 +36,151 @@ def set_para(gdf, selection, sliders, layer):
+                & (gdf[k] < v.value[1])
+                &(gdf[min_iso]) == True
+                &(gdf[max_iso]) == True
+                ]
+            select_outside = gdf[k].loc[
+                (gdf[k] >= v.value[0]) 
+                & (gdf[k] < v.value[1])
+                &(gdf[min_iso]) == False
+                &(gdf[max_iso]) == False
+                ]
         str_ += """
                 <tr>
-                  <th>% of roofs <i>{}</i> filter</th>
+                  <th>% of rooftops filtered by <i>{}</i></th>
                   <th>{} <= x < {}</th>
                   <th><b>{} %</b></th>
+                  <th><b>{} %</b></th>
+                  <th><b>{} %</b></th>
                 </tr>
                 """.format(
                 k,
                 round(v.value[0]),
                 round(v.value[1]),
-                round(
-                        len(
-                                gdf.loc[
-                                        (gdf[k] >= v.value[0]) 
-                                        & (gdf[k] < v.value[1])
-                                        ]
-                                ) / nb_roofs * 100
-                        )
+                round(len(selected) / nb_roofs * 100),
+                round(len(select_inside) / nb_roofs * 100),
+                round(len(select_outside) / nb_roofs * 100)
                 )
     
+    if min_iso == "0": 
+        selection_inside = gdf.loc[gdf[max_iso] == True]
+        selection_outside = gdf.loc[gdf[max_iso] == False]
+    else:
+        selection_inside = gdf.loc[
+            (gdf[min_iso] == True) 
+            & (gdf[max_iso] == True)
+            ]
+        selection_outside = gdf.loc[
+            (gdf[min_iso] == False) 
+            & (gdf[max_iso] == False)
+            ]
+                                
     str_ += """
              <tr>
-                  <th>Number of roofs with potential</th>
-                  <th> ----------- </th>
+                  <th>Number of filtered roofs</th>
+                  <th></th>
+                  <td><b>{}</b></td>
                   <td><b>{}</b></td>
+                  <td><b>{}</b></td>
+                  
              </tr>
             """.format(
-            len(selection)
+            len(selection),
+            len(selection_inside),
+            len(selection_outside)
             )
     str_ += """
              <tr>
-                  <th>Total number of roofs of the layer </th>
-                  <th> ----------- </th>
+                  <th>Total number of layer's rooftops</th>
+                  <th></th>
                   <td><b>{}</b></td>
              </tr>
             """.format(
             nb_roofs
             )
     str_ += """
              <tr>
-                  <th>% of roofs with potential </th>
-                  <th> ----------- </th>
+                  <th>% of rooftops with potential</th>
+                  <th></th>
+                  <td><b>{}%</b></td>
```

Comment:
I think it will be interesting to have more details on these 3 numbers.
1. selection
2. inner
3. outer 
etc.